app/renderer/components/main/findbar.js

```diff
@@ -275,16 +279,15 @@ class FindBar extends React.Component {
           id='caseSensitivityCheckbox'
           checkedOn={this.props.isCaseSensitive}
           onClick={this.onCaseSensitivityChange}
-        />
-        <label
-          htmlFor='caseSensitivityCheckbox'
```

Comment:
The label is now being rendered by the switchControl, and the findBarTestStyle style overrides have been changed to aphrodite-style findBarTextStyles css.